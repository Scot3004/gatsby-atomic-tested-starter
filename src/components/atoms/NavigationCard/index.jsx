import styled from 'styled-components';
import Card from '../Card';

export const NavigationCard = styled(Card)`
  padding: 0;
`;

export default NavigationCard;
