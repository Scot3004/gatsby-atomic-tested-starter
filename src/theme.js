const theme = {
  backgroundColor: '#f5f5f5',
  textColor: '#000',
  mutedColor: '#555',
  primaryColor: '#663399',
  visitedColor: '#402060'
};

export default theme;
