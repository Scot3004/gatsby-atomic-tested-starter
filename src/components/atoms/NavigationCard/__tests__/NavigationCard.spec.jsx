import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import theme from '../../../../theme';
import NavigationCard from '..';

describe('When I want to see the navigation elements', () => {
  const tree = renderer.create(<NavigationCard theme={theme} />).toJSON();
  test('Then they should not have space from the parent', () => {
    expect(tree).toMatchSnapshot();
  });
});
