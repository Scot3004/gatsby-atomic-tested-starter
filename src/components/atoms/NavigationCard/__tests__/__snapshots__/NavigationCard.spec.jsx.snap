// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When I want to see the navigation elements Then they should not have space from the parent 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border-radius: 0.5rem;
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
  box-sizing: border-box;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin: 0 0 1rem 0;
  padding: 0.6rem;
  -webkit-transition: 0.3s;
  transition: 0.3s;
  width: 100%;
  padding: 0;
}

.c0:last-child {
  margin: 0;
}

<div
  className="c0"
/>
`;
